sympy/integrals/__init__.py

```diff
@@ -9,7 +9,11 @@
     >>> integrate(sin(x),x)
     -cos(x)
 """
+from .deltafunctions import deltaintegrate
+from .heurisch import heurisch, heurisch_wrapper
 from .integrals import integrate, Integral, line_integrate
+from .manualintegrate import manualintegrate, integral_steps
+from .rationaltools import ratint
```

Comment:
On Mon, Mar 09, 2015 at 06:09:10AM -0700, Joachim Durchholz wrote:
>    As I can see in the stable version, the docs examples are correct, not
>    wrong:

Yes, because they don't have "shortened" imports.

PS: Actually, this all was a note for your reviewer, not for you.